### PR TITLE
ISSITES-5743/ISSITES-5744 public sites nav changes

### PR DESCRIPTION
**Pull request by liaoethan**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/fragments/group/public-sites-navigation/fragments/public-sites-navigation/index.js [hunk 1]
```diff
@@ -11,3 +11,31 @@
  * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
  * details.
  */
+
+const menuButtonGroup = document.querySelector('.menu-button-group');
```
Comment (allen-ziegenfus) [inline modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/fragments/group/public-sites-navigation/fragments/public-sites-navigation/index.js:15]: @liaoethan  These selectors are pretty non-specific to use on the whole document. You can instead just add `/* eslint-disable no-undef */` to ignore the fragmentElement error

File: modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/style-books/learn/frontend-tokens-values.json [hunk 2]
```diff
@@ -12,6 +12,11 @@
 		"name": "brandColor1",
 		"value": "#0b5fff"
 	},
+	"btnOutlineHoverColor": {
+		"cssVariableMapping": "btn-outline-hover-color",
```
Comment (allen-ziegenfus) [inline modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/style-books/learn/frontend-tokens-values.json:16]: This token does not exist in dialect when I searched. There are a lot of different hover tokens in dialect (48 tokens) did you not find any there you could use?

**Review by allen-ziegenfus — CHANGES_REQUESTED** [review]

**Comment by liaoethan** [comment]
Thanks @allen-ziegenfus, I made changes to the two suggestions you made above

**Comment by allen-ziegenfus** [comment]
thanks @liaoethan  I had to squash this since there were too many commits: https://github.com/brianchandotcom/liferay-portal/pull/124843

**Pull request closed without merging**